lib/cli.rb

```diff
@@ -533,6 +536,9 @@ def self.supports_filtering(command)
       if options["extract-files"] || options["extract-unmanaged-files"]
         inspect_options[:extract_unmanaged_files] = true
       end
+      if options["docker-container"]
+        inspect_options[:docker_container] = true
```

Comment:
Why is the option treated as a boolean option here, when it is defined as a string option before?